app/uk/gov/hmrc/incometaxsoftwarechoicesfrontend/views/templates/SoftwareVendorsAgentTemplate.scala.html

```diff
@@ -0,0 +1,35 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.viewmodels.SoftwareChoicesResultsViewModel
+@import uk.gov.hmrc.incometaxsoftwarechoicesfrontend.views.html.templates.SoftwareVendorsTemplate
+
+@this(
+softwareVendorsTemplate: SoftwareVendorsTemplate
+)
+
+@(viewModel: SoftwareChoicesResultsViewModel)(implicit messages: Messages)
+
+<div id="vendor-count">
+    <h2 class="govuk-heading-l">@messages("search-software.vendor-count.agent.heading", viewModel.otherVendors.vendors.length)</h2>
+    <h3 class="govuk-heading-m">
+        @messages("search-software.vendor-count.agent")
+    </h3>
```

Comment:
In the protoype this is a paragraph - 'You can use the filter options to narrow down your search.'

 There should also be a divider line below this